eng/common/scripts/modules/Package-Properties.psm1

```diff
@@ -3,11 +3,12 @@
 class PackageProps
 {
     [string]$pkgName
-    [AzureEngSemanticVersion]$pkgVersion
```

Comment:
Why change to a string? I was expecting we could loosen up our SemanticVersion type instead to make some of the pre-release versions optional.

Reply:
I filled an [issue](https://github.com/Azure/azure-sdk-tools/issues/770) to take car of that, and it think its best to work on that after settling on an standard way for naming previews in all languages. Am still parsing it using the class but the `ToString()` method on the class only works for dotnet versions. Also I cannot think of a scenarios where the distinct components of the versions becomes needed.